conftest.py

```diff
@@ -81,10 +81,8 @@ def small_cluster(request):
     with Cluster(
         name=f"{module}-{uuid.uuid4().hex[:8]}",
         n_workers=10,
-        worker_memory="8 GiB",
-        worker_vm_types=["m5.large"],
-        scheduler_vm_types=["m5.large"],
-        scheduler_options={"idle_timeout": "1 hour"},
```

Comment:
I'm noticing we don't have the scheduler timeout. Do we need/want it?. I believe this change is part of solving a conflict with main but taking the opportunity to think about whether we need this or not. 

I believe we don't as the clusters, in this case, won't be idle.

Reply:
That's correct. I bumped the idle timeout up to an hour as a workaround for https://github.com/dask/distributed/issues/6494, but as that issue has been closed in main now, I think we can revert back to using the default 20 minute idle timeout